app/src/main/java/org/mozilla/focus/utils/FileUtils.java

```diff
@@ -185,4 +186,18 @@ public static long clearCache(Context context) {
         WebStorage.getInstance().deleteAllData();
         return FileUtils.deleteWebViewCacheDirectory(context);
     }
+
+    public static void writeBundleToStorage(@NonNull final File dir,
+                                            @NonNull final String fileName,
+                                            @NonNull final Bundle bundle) {
+        ensureDir(dir);
+        // TODO: Not implement yet
+    }
+
+    public static Bundle readBundleFromStorage(@NonNull final File dir,
+                                               @NonNull final String fileName) {
+        ensureDir(dir);
+        // TODO: Not implement yet
+        return null;
+    }
```

Comment:
In my personal view, 'ToStorage' and 'FromStorage' might be redundant, but keep it still OK to me.

Is this method designed to write/read general Bundle or specific to WebViewState only?

Reply:
They are general methods for persisting Bundle in the long run.